src/cosmos/authz/v1beta1/tx.ts

```diff
@@ -1,98 +1,109 @@
 /* eslint-disable */
-import Long from "long";
-import _m0 from "protobufjs/minimal";
-import { Any } from "../../../google/protobuf/any";
 import { Grant } from "./authz";
-
+import { Any } from "../../../google/protobuf/any";
+import * as _m0 from "protobufjs/minimal";
+import { isSet, DeepPartial, Exact, bytesFromBase64, base64FromBytes, Rpc } from "../../../helpers";
 export const protobufPackage = "cosmos.authz.v1beta1";
-
-/** Since: cosmos-sdk 0.43 */
-
 /**
  * MsgGrant is a request type for Grant method. It declares authorization to the grantee
  * on behalf of the granter with the provided expiration time.
  */
+
 export interface MsgGrant {
   granter: string;
   grantee: string;
-  grant?: Grant;
+  grant: Grant;
 }
-
 /** MsgExecResponse defines the Msg/MsgExecResponse response type. */
+
 export interface MsgExecResponse {
   results: Uint8Array[];
 }
-
 /**
  * MsgExec attempts to execute the provided messages using
  * authorizations granted to the grantee. Each message should have only
  * one signer corresponding to the granter of the authorization.
  */
+
 export interface MsgExec {
   grantee: string;
   /**
    * Authorization Msg requests to execute. Each msg must implement Authorization interface
    * The x/authz will try to find a grant matching (msg.signers[0], grantee, MsgTypeURL(msg))
    * triple and validate it.
    */
+
   msgs: Any[];
 }
-
 /** MsgGrantResponse defines the Msg/MsgGrant response type. */
-export interface MsgGrantResponse {}
 
+export interface MsgGrantResponse {}
 /**
  * MsgRevoke revokes any authorization with the provided sdk.Msg type on the
  * granter's account with that has been granted to the grantee.
  */
+
 export interface MsgRevoke {
```

Comment:
This whitespacing looks a bit strange. Shouldn't the newline be above the doc comment instead of below?